app/workers/functions.php

```diff
@@ -217,6 +221,11 @@
                 ->setParam('networkResponseSize', 0)
                 ->submit();
         }
+
+        if (!empty($error)) {
+            $projectId = $project->getId();
+            throw new Exception("Error in project #{$projectId}, function #{$functionId}: " . $error);
```

Comment:
I'm not sure this the best way to utilize Sentry. It will treat each error as its own separate things. Can we make our exception accept more metadata and them pass it properly to Sentry or any other logging provider?

Reply:
Made log a resource and updated it from worker